.vib/gitea/cypress/cypress.json

```diff
@@ -0,0 +1,8 @@
+{
+  "env": {
+    "username": "user@example.com",
+    "password": "bitnami!1234"
+  },
+  "baseUrl": "http://localhost",
+  "defaultCommandTimeout": 60000
```

Comment:
Minor nitpick, but we can probably reduce this to 30s as gitea seems lightweight (by convention, we use increments of 30s).